### ISSUE TITLE
[Bug] [a11y] Card number and name edit boxes do not have an a11y label

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/fenix#19419]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have the exact bug and steps to reproduce described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

## Steps to reproduce

Open "Add card" screen with Talkback on.

### Expected behavior

Input edit boxes should have a label as a hint to what is expected to be inputted by the user.

### Actual behavior

Input edit boxes do not have a label as a hint to what is expected to be inputted by the user. They just have a text box that serves as a label just vissually.

### Device information

* Device vendor / model and Android version: ?
* Firefox for Android version: ? (go to Settings -> About Firefox)
 
Note: The  edit credit card screen has the same issue.

**Comment by priyanka0906** [comment]
@mcarare may I work on this.

**Comment by mcarare** [comment]
@priyanka0906 We should wait for UX feedback on this.

**Comment by priyanka0906** [comment]
> @priyanka0906 We should wait for UX feedback on this.

@mcarare okay.

**Comment by amedyne** [comment]
Tagging @topotropic

**Comment by topotropic** [comment]
We should do an audit of our forms and align them – that might take a moment. I'll add it to my list.

**Comment by mcarare** [comment]
Added a quick fix for this in https://github.com/mozilla-mobile/fenix/pull/19762. The present issue is independent of the UX task related to forms investigation, so that issue will be tracked in https://github.com/mozilla-mobile/fenix/issues/19763.

**Comment by abodea** [comment]
Verified as fixed on the Nightly 6/4 with Google Pixel 4 XL (Android 11).
Note that the `Card Number and Name on Card` edit box are read correctly.
